media/utils.py

```diff
@@ -54,9 +56,17 @@ def oembed_from_id(object_id, oembed_url, service, field, pattern):
             _("Could not validate %(field)s due to invalid response from %(service)s.")
             % locals()
         ) from e
+    
+    # Youtube asset doesn't have this attribute
+    track_id = None
+    track_id_search = re.search("tracks%2F(\d+)", data.get('html', ''))
+    if track_id_search:
+        track_id = track_id_search.group(1)
+
 
     return OEmbedData(
         object_id=object_id,
         title=data.get("title"),
         thumbnail_url=data.get("thumbnail_url"),
+        track_id=track_id or None,
```

Comment:
`track_id` is already defaulted above to `None`:

```suggestion
        track_id=track_id,
```